dss_datamover/utils/config.py

```diff
@@ -57,7 +57,8 @@ def process_config(self):
             config = json.loads(cfg.read().decode('UTF-8', "ignore"))
         if self.params:
             for param in self.params:
-                config[param] = self.params[param]
+                if self.params[param] is not None:  # must explicitly check for None since we still want False values to be overriden onto the config dict
```

Comment:
`is not None` is redundant in python

Reply:
I have to explicitly check for none, so the following does not work
`if self.params[param]:` 

I could change the logic to the following, but I don't think it is necessary.
```
if self.params[param] is None:
  continue
```